ingestion/src/metadata/ingestion/sink/metadata_rest.py

```diff
@@ -230,7 +221,13 @@ def write_dashboard_usage(self, dashboard_usage: DashboardUsage) -> None:
             logger.debug(traceback.format_exc())
             logger.error(f"Failed to write dashboard usage [{dashboard_usage}]: {exc}")
 
-    def write_tables(self, db_schema_and_table: OMetaDatabaseAndTable):
+    def write_tables(  # pylint: disable=too-many-branches
```

Comment:
This we need to deprecate, opened https://github.com/open-metadata/OpenMetadata/issues/7960